config/initializers/business_time.rb

```diff
@@ -0,0 +1,28 @@
+  require 'holiday_calendar'
+
+  module BusinessTimeHelpers
+    extend self
+
+    def holidays
+      scope
+        .to_a
+        .reduce([]) { |hols, y| hols << calendar.list_for_year(y) }
+        .flatten
+        .map { |h| Date.parse(h) } 
+    end
+
+    def scope
+      (((Date.today - PqStatistics::WINDOW.days).year - 1)..(Date.today.year + 1))
```

Comment:
No need for the extra parentheses
(Date.today - PqStatistics::WINDOW.days).year - 1..Date.today.year + 1 works but 
((Date.today - PqStatistics::WINDOW.days).year - 1)..(Date.today.year + 1) probably better for clarity